packages/block-editor/src/store/reducer.js

```diff
@@ -717,8 +717,21 @@ const BLOCK_SELECTION_INITIAL_STATE = {
  */
 export function blockSelection( state = BLOCK_SELECTION_INITIAL_STATE, action ) {
 	switch ( action.type ) {
-		case 'CLEAR_SELECTED_BLOCK':
+		case 'WIPE_SELECTED_BLOCK':
```

Comment:
It would be great to cover `WIPE_SELECTED_BLOCK` and `RESTORE_SELECTED_BLOCK` reducer updates with unit tests.

Reply:
added the tests!

Reply:
Those are tests for actions, they don't test the reducer :)
Tests for actions aren't that helpful as those are simple wrappers on static objects.